src/_live-rendering.ts

```
@@ -181,17 +182,17 @@ export function renderLiveDisplay(terminal: TerminalManager, block: SessionBlock
-				totalTokens,
-				config.tokenLimit,
-				barWidth,
-				{
-					showPercentage: false,
-					fillChar: barColor('█'),
-					emptyChar: pc.gray('░'),
-					leftBracket: '[',
-					rightBracket: ']',
-				},
-			)
+			totalTokens,
```

ESLint

[error] Expected indentation of 4 tabs but found 3.

(style/indent)